projects/packages/forms/src/contact-form/libs/date-picker/date-picker-options.ts

```diff
@@ -0,0 +1,149 @@
+/**
+ * @file Responsible for sanitizing and creating date picker options.
+ */
+import { IDatePicker, IDatePickerOptions } from './interfaces';
+import { now, shiftYear, dateOrParse, Dec31st, Jan1st } from './lib/date';
+import { cp } from './lib/fns';
+
+const english = {
+	days: [ 'Su', 'Mo', 'Tu', 'We', 'Th', 'Fr', 'Sa' ],
+	months: [
+		'January',
+		'February',
+		'March',
+		'April',
+		'May',
+		'June',
+		'July',
+		'August',
+		'September',
+		'October',
+		'November',
+		'December',
+	],
+	today: 'Today',
+	clear: 'Clear',
+	close: 'Close',
+	ariaLabel: {
+		enterPicker:
+			'You are on a date picker input. Use the down key to focus into the date picker. Or type the date in the format MM/DD/YYYY',
+		dayPicker:
+			'You are currently inside the date picker, use the arrow keys to navigate between the dates. Use tab key to jump to more controls.',
+		monthPicker:
+			'You are currently inside the month picker, use the arrow keys to navigate between the months. Use the space key to select it.',
+		yearPicker:
+			'You are currently inside the year picker, use the up and down arrow keys to navigate between the years. Use the space key to select it.',
+		monthPickerButton: 'Month picker. Use the space key to enter the month picker.',
+		yearPickerButton: 'Year picker. Use the space key to enter the month picker.',
+		dayButton: 'Use the space key to select the date.',
+		todayButton: 'Today button. Use the space key to select the current date.',
+		clearButton: 'Clear button. Use the space key to clear the date picker.',
+		closeButton: 'Close button. Use the space key to close the date picker.',
+	},
+};
+
+/**
+ * DatePickerOptions constructs a new date picker options object, overriding
+ * default values with any values specified in opts.
+ *
+ * @param  _options
+ *
+ * @returns {IDatePickerOptions}
+ */
+export function DatePickerOptions( _options: Partial< IDatePickerOptions > = {} ) {
+	const options = cp( defaults(), _options );
```

Comment:
I think this would fix those checks:
```suggestion
	const options: Partial< IDatePickerOptions > = cp( defaults(), _options );
```